src/components/ADempiere/TabManager/AdvancedTabQuery.vue

```diff
@@ -333,6 +333,17 @@ export default defineComponent({
         filters,
         isAdvancedQuery: true
       })
+        .then(response => {
+          const currentRoute = router.app._route
+          const { query } = currentRoute
+          const { tableName } = store.getters.getStoredTab(props.parentUuid, props.containerUuid)
+          router.replace({
+            query: {
+              ...query,
+              recordId: response[0][tableName + '_ID']
```

Comment:
Please evaluate if the record is empty, and the action evaluates which is the uuid of the record to be established as current, the set in the path should be added there

Reply:
https://github.com/solop-develop/frontend-core/pull/1868/commits/7e237171b3b0ee774ece67a8c78b40f79b97b32c